* add option to respect AR
 (respecting PKG_CONFIG is handled in the ebuild)
* fix icu unbundling underlinking
* handle recastnavigation unbundling
--- a/SConstruct
+++ b/SConstruct
@@ -233,4 +233,5 @@
 opts.Add("CXX", "C++ compiler")
 opts.Add("CC", "C compiler")
+opts.Add("AR", "Archiver")
 opts.Add("LINK", "Linker")
 opts.Add("CCFLAGS", "Custom flags for both the C and C++ compilers")
--- a/platform/linuxbsd/detect.py
+++ b/platform/linuxbsd/detect.py
@@ -238,5 +238,5 @@
 
     if not env["builtin_icu"]:
-        env.ParseConfig("pkg-config icu-uc --cflags --libs")
+        env.ParseConfig("pkg-config icu-i18n --cflags --libs")
 
     if not env["builtin_harfbuzz"]:
@@ -249,4 +249,7 @@
         env.ParseConfig("pkg-config libenet --cflags --libs")
 
+    if not env["builtin_recast"]:
+        env.ParseConfig("pkg-config recastnavigation --cflags --libs")
+
     if not env["builtin_squish"]:
         env.ParseConfig("pkg-config libsquish --cflags --libs")
